### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by wiersgallak**
Update Marketplace submission process

### CODE REVIEW

**Review by esethna — APPROVED** [review]

**Comment by esethna** [comment]
@cwarnermm can you help point us in the right direction for how to set up a redirect from https://developers.mattermost.com/integrate/plugins/community-plugin-marketplace/ (we want to delete this page) to this updated page (https://developers.mattermost.com/integrate/marketplace-submissions/)

File: site/content/integrate/marketplace-submissions/_index.md [hunk 1]
```diff
@@ -7,38 +7,34 @@ aliases:
...
 
-Please fill out the information in {{<newtabref title="this form" href="https://forms.gle/PE8kmfSuneP9GWnq8">}} to be reviewed by a member of the Mattermost team.
+Please fill out the information in {{<newtabref title="this form" href="https://forms.gle/PX6BwUF1QEkRjxDw7">}} to be reviewed by a member of the Mattermost team.
 
 Every contribution goes through the following checklist before being added to the web Marketplace at {{<newtabref title="mattermost.com/marketplace" href="https://mattermost.com/marketplace">}}:
 
 1. There is a link to the contribution so we can test it out.
 2. There is documentation to support people installing, configuring, and using the contribution. We also recommend including screenshots to give users a better understanding of the workflow and user experience.
 3. There is a public issue or bug tracker for users to report bugs or issues they encounter.
+4. You've included a link to an image for your contribution
```
Comment (cwarnermm) [inline site/content/integrate/marketplace-submissions/_index.md:28]:
```suggestion
4. You've included a link to an image for your contribution:
```

**Comment by cwarnermm** [comment]
Hi @esethna @wiersgallak! 

I recommend that, instead of removing the page (``https://developers.mattermost.com/integrate/plugins/community-plugin-marketplace/``), replace the content with text and a link to the new page. 

- When users reach this page via the navigation pane, they'll see a CTA to take them to the new page.
- Dev docs redirects via Hugo work differently than product docs redirects via Sphinx. Dev docs redirects are backward redirects. This means that the "old" page must exist in the nav pane tree in order for the auto-redirect to work. Removing the old nav page will break the redirect. That's why I suggest clearing the old page and replacing it with a simple CTA.
- Users who enter ``https://developers.mattermost.com/integrate/plugins/community-plugin-marketplace/`` URL into a browser are automatically taken to the updated page. This is due to the ``Alias`` metadata located at the top of the new page. (You can try this now in production.)

If this workaround isn't ideal, we'd need to research whether it's possible to set up forward redirects via Hugo. It's not something we've tried before, and Hugo may/may not even support it without extra effort/code.

**Comment by esethna** [comment]
@cwarnermm sounds fine to me. In that case I'd recommend we replace the content on that page with something like this:

```
Community plugins in the Marketplace

Once your plugin has reached a certain level of quality, you might consider submitting it to the web Marketplace. The {{<newtabref title="Mattermost web Marketplace" href="https://mattermost.com/marketplace/">}} supports discovery, installation, and updates of extensions of the Mattermost platform. If you have built a plugin, app, playbook, integration, or tool that helps other users get more from Mattermost, the web Marketplace is a great way to get feedback on your contribution and help make it more popular. 

Read more about the process at https://developers.mattermost.com/integrate/marketplace-submissions/#
```

**Comment by cwarnermm** [comment]
@wiersgallak - This PR is ready for your final review. The following page has been updated: http://mattermost-dev-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/1255/integrate/plugins/community-plugin-marketplace/

**Comment by wiersgallak** [comment]
Thanks @cwarnermm.  Approved from my perspective.

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**